index.html

```diff
@@ -703,7 +703,14 @@ <h3>Connect With Us!</h3>
                       <a href="mailto:eclub.usfca@gmail.com" title="Email" target="_blank"><i class="fa fa-envelope"></i></a>
                       <a href="https://www.instagram.com/usfcaeclub/" title="Instagram" target="_blank"><i class="fa fa-instagram"></i></a>
                   </div>
-                      
+                     
+                  <!--Google Map of School Location-->                
+                  <div>
+                    <p></p>
+                    <center><iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3153.6186677684814!2d-122.45187968456115!3d37.7755386797594!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8085874c5e815e35%3A0xcb33a474306259e9!2sMalloy+Hall%2C+USF+Main+Campus!5e0!3m2!1sen!2sus!4v1552096197917" width="750" height="300" frameborder="0" style="border:0" allowfullscreen></iframe></center>
```

Comment:
I like what you have so far, I'm thinking a 2 column footer might look better though.